controller/src/beerocks/master/tasks/client_steering_task.cpp

```diff
@@ -75,8 +75,6 @@ void client_steering_task::work()
                 database.update_node_failed_24ghz_steer_attempt(sta_mac);
             }
         }
-
-        finish();
```

Comment:
Why did you remove it? This line has ended the task.

Reply:
you are right I'll fix that.